.gitpod.Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM gitpod/workspace-full@sha256:bb84c4490170fbfdeb577535a6dec5aafd1eb8fdf433f9a344bd8f4a69af5707
+ARG FNSA_VERSION=2.0.1
+
+RUN rustup toolchain install 1.69.0
+RUN rustup default 1.69.0
+RUN rustup target add wasm32-unknown-unknown
+RUN rustup default 1.74.1
```

Comment:
It seems to be set to 1.69.0 by default, but do we also need this?

Reply:
Because cargo-generate v0.19.0 need the latest version of rustc.

Reply:
https://docs.finschia.network/smart-contract/environment-setup
According to the docs,  Rust 1.69.0 or lower is supported so 1.69.0 is set as default.

Reply:
1.74.1 is required to build cargo-check and cargo-generation.